extensions/amp-story/1.0/amp-story-interactive-poll.css

```diff
@@ -76,12 +73,19 @@
   transform: translateX(calc(100% - var(--option-percentage)))!important;
 }
 
+.i-amphtml-story-interactive-poll-container:not(.i-amphtml-story-interactive-has-data) .i-amphtml-story-interactive-option::after {
+  display: none !important;
+}
```

Comment:
Removed the percentage bars on quizzes if no endpoint, fixing the offline post-select state